### ISSUE TITLE
v0.5.3: ERROR mainline::rpc: Could not bootstrap the routing table

### ISSUE THREAD

**Issue by HolgerHatGarKeineNode** [pubky/pkdns#28]
I get this error with the new version: v.0.5.3

I did nothing more than use the new binary. And the system is simply a bare Debian 6.1.112-1

`ERROR mainline::rpc: Could not bootstrap the routing table`

And it looks like I can't resolve pkarr addresses anymore. The resolution now somehow appends other TLDs from my provider network. I have masked the name of the provider with ‘x’.

```
2024-11-07T16:27:48.019316Z DEBUG {query_id=18093}: pkdns::pkarr_resolver: New query: 7fmjpcuuzf54hw18bsgi3zihzyh4awseeuq5tmojefaezjbd64cy.xx.xxxx.network TYPE(A)
2024-11-07T16:27:48.019532Z DEBUG {query_id=18093}: pkdns::pkarr_resolver: TLD .network is not a pkarr key. Fallback to ICANN. 
2024-11-07T16:27:48.036279Z DEBUG {query_id=18093}: any_dns::dns_socket: Processed query 7fmjpcuuzf54hw18bsgi3zihzyh4awseeuq5tmojefaezjbd64cy.xx.xxxx.network TYPE(A) within 16ms
2024-11-07T16:27:48.037463Z DEBUG {query_id=16350}: pkdns::pkarr_resolver: New query: 7fmjpcuuzf54hw18bsgi3zihzyh4awseeuq5tmojefaezjbd64cy.xx.xxxx.network TYPE(AAAA)
2024-11-07T16:27:48.037576Z DEBUG {query_id=16350}: pkdns::pkarr_resolver: TLD .network is not a pkarr key. Fallback to ICANN. 
2024-11-07T16:27:48.052837Z DEBUG {query_id=16350}: any_dns::dns_socket: Processed query 7fmjpcuuzf54hw18bsgi3zihzyh4awseeuq5tmojefaezjbd64cy.xx.xxxx.network TYPE(AAAA) within 15ms
```

**Comment by SeverinAlexB** [comment]
Interesting. Did you set pkdns as your system dns on the server you are running it on by any chance?

**Comment by SeverinAlexB** [comment]
I fixed #24. https://github.com/pubky/pkdns/releases/tag/v0.5.4 might work for you now.
